Dockerfile.rootless

```diff
@@ -21,6 +21,8 @@ WORKDIR ${GOPATH}/src/code.gitea.io/gitea
 #Checkout version if set
 RUN if [ -n "${GITEA_VERSION}" ]; then git checkout "${GITEA_VERSION}"; fi \
  && make clean-all build
+ && chmod 755 gitea
+ && chown root:root gitea
```

Comment:
Probably needs some `\`. Generally I recommend this syntax:

```
cmd && \
  cmd && \
  cmd
```

Reply:
yes, right, I did just more like a proof of concept PR to get opinions, did even directly from GH web :]
fixed :)